docs/team/theopin.md

```diff
@@ -1,6 +1,85 @@
-# Theodore Pinto - Project Portfolio Page
+# Theodore Pinto's Project Portfolio Page 
 
 ## Overview
```

Comment:
Hmm I think the overview is meant to be overview of yourself instead of the program

Reply:
From the module website it says this tho: A short overview of your product to provide some context to the reader.